packages/connectivity/src/scp-cf/destination/service-binding-to-destination.ts

```diff
@@ -56,6 +56,24 @@ export async function transformServiceBindingToDestination(
   );
 }
 
+/**
+ * Transforms a service binding to a destination of type OAuth2ClientCredentials.
```

Comment:
[pp] Can we also add some hint on when to use this over  `transformServiceBindingToDestination`?

I also find this description from the other function helpful:

```
Convenience function to create a destination from the provided service binding.
 * If a JWT is provided as part of options, the tenant in the JWT is used for client credentials grant, else the provider tenant is used, wherever applicable.
```

Reply:
Yeah that's a good idea, I wasn't sure if putting it in our docs is enough or if I should put it here as well.
I'll add it!